docs/getting-started/setup/ui-language.md

```diff
@@ -41,136 +42,79 @@ See the demo of the editor in Arabic:
 
 We are doing our best to deliver the best RTL support to our users and we constantly improve the editor. Check out the [RTL support](https://github.com/ckeditor/ckeditor5/issues/1151) issue on GitHub to learn more and stay up–to–date. Your feedback is much appreciated!
 
-## Loading additional languages from CDN, npm, and a zip file
+## Loading additional languages from npm or CDN
 
  By default, the editor will display in American English. This is the language built into the `ckeditor.js` files. To change the language of the editor UI, you need to load additional language file(s). Check out the following sections to see how to do that:
 
-* [CDN](#cdn),
-* [npm](#npm),
-* [Zip download](#zip).
+* [npm](#npm)
+* [CDN](#cdn)
 
-Next, you can configure the editor to use the chosen language:
+### npm
+
+After installing the build from npm, languages will be available via `ckeditor5/dist/translations/[lang].js`.
+
+You can load a single language directly to your code by importing. For example, to use Polish, call `'ckeditor5/dist/translations/pl.js'`. Sample editor code will look similar to this one:
 
 ```js
-ClassicEditor
-	.create( document.querySelector( '#editor' ), {
-		// The language code is defined in the https://en.wikipedia.org/wiki/ISO_639-1 standard.
-		language: 'es'
-	} )
-	.then( editor => {
-		console.log( editor );
-	} )
-	.catch( error => {
-		console.error( error );
-	} );
+import { ClassicEditor, Essentials, Paragraph } from 'ckeditor5';
+import translations from 'ckeditor5/dist/translations/pl.js';
+
+import 'ckeditor5/dist/styles.css';
+
+await ClassicEditor.create( document.querySelector( '#editor' ), {
+	plugins: [
+		Essentials,
+		Paragraph,
+	],
+	toolbar: {
+		items: [ 'undo', 'redo' ]
+	},
+	translations
+} );
 ```
 
+See the {@link getting-started/quick-start#installing-ckeditor-5-using-npm npm installation guide} for more information.
+
 ### CDN
 
-To use a different language than the default one (English), you need to load the editor together with the preferred language:
+To use different language than the default one (English), you need to load the editor together with the preferred language:
 
 ```html
-<script src="https://cdn.ckeditor.com/ckeditor5/[version.number]/[distribution]/ckeditor.js"></script>
-<script src="https://cdn.ckeditor.com/ckeditor5/[version.number]/[distribution]/translations/[lang].js"></script>
+<script src="https://cdn.ckeditor.com/ckeditor5/dist/ckeditor.js"></script>
+<script src="https://cdn.ckeditor.com/ckeditor5/dist/translations/[lang].js"></script>
 ```
```

Comment:
These two tags will no longer be required.

Reply:
OK, should I just delete this passage then?